scripts/check_md5sums.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ] || ! { [ "$1" = "zson" ] || [ "$1" = "bzson" ]; }; then
+  echo 'Must specify output type to be cheecked: "zson" or "bzson"'
```

Comment:
s/cheecked/checked/

nit: i think we tend to say "output format" more than "output type"

Reply:
Just pointing out there is also "log format" above.

Reply:
Ok, I fixed the typo, and opted for "output format".